common/src/main/java/org/conscrypt/NativeSsl.java

```diff
@@ -663,13 +663,29 @@ int getPendingWrittenBytes() {
         }
```

Comment:
Should we use `lock.readLock()` in the check in `getPendingWrittenBytes` ?

Reply:
Yes, please.